index.html

```diff
@@ -0,0 +1,244 @@
+            no accounts or sign-ups required.
+          </div>
+          <div>
+            <ul class="flex">
+              <li>
+                html
+              </li>
+              <li>
+                css
+              </li>
+              <li>
+                javascript
+              </li>
+            </ul>
+          </div>
+          <div>
+            <button> See Project</button>
+          </div>
+        </div>
+
+        <div class="grid column row">
+          <div class="two"></div>
+          <div>Multi-Post Stories</div>
+          <div>
+            <div>CANOPY</div>
+            <ul class="flex">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <div>
+            A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+          </div>
+          <div>
+            <ul class="flex">
+              <li>
+                html
+              </li>
+              <li>
+                css
+              </li>
+              <li>
+                javascript
+              </li>
+            </ul>
+          </div>
+          <div>
+            <button> See Project</button>
+          </div>
+        </div>
+
+        <div class="grid column row">
+          <div class="three"></div>
+          <div>Tonic</div>
+          <div>
+            <div>CANOPY</div>
+            <ul class="flex">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <div>
+            A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+          </div>
+          <div>
+            <ul class="flex">
+              <li>
+                html
+              </li>
+              <li>
+                css
+              </li>
+              <li>
+                javascript
+              </li>
+            </ul>
+          </div>
+          <div>
+            <button> See Project</button>
+          </div>
+        </div>
+
+        <div class="grid column row">
+          <div class="four"></div>
+          <div>Multi-Post Stories</div>
+          <div>
+            <div>CANOPY</div>
+            <ul class="flex">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <div>
+            A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+          </div>
+          <div>
+            <ul class="flex">
+              <li>
+                html
+              </li>
+              <li>
+                css
+              </li>
+              <li>
+                javascript
+              </li>
+            </ul>
+          </div>
+          <div>
+            <button> See Project</button>
+          </div>
+        </div>
+
+        <div class="about grid row2 column">
+          <section>
+            <h1>About<br>Myself</h1>
```

Comment:
you forgot to add the font weight to your title please try to stick to the Figma design `font-weight: 700;`